src/components/CardForm/CardForm.js

```diff
@@ -59,6 +59,7 @@ const CardForm = ({ handleCancelButton, handleAddCard }) => {
 };
 
 CardForm.propTypes = {
+  colId: PropTypes.number.isRequired,
```

Comment:
Maybe you could rename that variable from `colId` to `columnId`. If you disagree on the naming then please ignore that suggestion.

Reply:
we are staying with calId, naming conflict